modules/src/ics07_tendermint/consensus_state.rs

```diff
@@ -50,7 +50,6 @@ impl DomainType<RawConsensusState> for ConsensusState {}
 impl TryFrom<RawConsensusState> for ConsensusState {
     type Error = Error;
 
-    // TODO: Fix height conversion below (which ignores epoch number, hence it's incorrect).
     // Related: https://github.com/informalsystems/ibc-rs/issues/191.
```

Comment:
I guess we can remove the issue mentioned in this comment.

Reply:
6b18f9f